src/rules/adaptive-mfa.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @title Adaptive MFA
```

Comment:
I think we should use a more specific name like "Adaptive MFA - Impossible Travel Only"

Reply:
@hernanhht 

So this template is meant to be an example of how to access the granular assessor results to determine a "custom" confidence score, and I've used Impossible Travel just to pick one out of the three we have. It's not like this is a use-case that we think customers will need/use, just a showcase of what's possible.

In that spirit, I think I prefer a more generic title, not specifically mentioning Impossible Travel, but to invite tenant admins to start from this example and work out their specific confidence calculation. Maybe we can be a more specific than just "Adaptive MFA". Ideas:

- Adaptive MFA with Custom Confidence (my preference)
- Adaptive MFA with Overridden Confidence
- Adaptive MFA with Special Confidence

wdyt?

Reply:
Ok, "with custom confidence" is good enough

Reply:
done